applications/MappingApplication/custom_utilities/mapper.h

```diff
@@ -182,15 +182,15 @@ class Mapper
 
     // Constructor, can only be called by derived classes (actual mappers)
     Mapper(ModelPart& rModelPartOrigin, ModelPart& rModelPartDestination,
-           Parameters JsonParameters) :
```

Comment:
Hi Riccardo,
Thanks for pointing this out, I didn't know.
So I should always pass the parameters by value (`Parameters JsonParameters`)? And in doing this I will create another shared pointer to the underlying data?